src/patterns/dosdp-patterns/response_to_chemical_stimulus_trait.yaml

```diff
@@ -0,0 +1,47 @@
+---
+pattern_name: response_to_chemical_stimulus_trait
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/response_to_chemical_stimulus_trait.yaml
+
+description: 'A trait that affects the biological response to a chemical.'
```

Comment:
`affects` does not seem like the right word here, but ok for now.. Just keep in mind to update the language moving forward.

Reply:
@matentzn How about
`A trait of a biological response to a chemical.`
? Would that wording be better?

Reply:
Again, I dont know whether "trait of a response" is the right grammar.. Just leave it for now, don't worry. Ask David in your next meeting with him

Reply:
I think 'affects' is good enough.  I can't think of a non-confusing alternative.  I think this comes down to the basic problem/confusion of 'phenotypic effect' vs 'abstract trait'/measurement.  The association with a variant in the GWAS catalog is certainly recording some effect on response to stimulus with the chemical in question.